### PR TITLE
README as fragments

### PR DESCRIPTION
**Pull request by sbidoul**
Since the README had a minor glitch that blocked the pypi upload I take the opportunity to convert it to fragments.

More important (to be reviewed):
* guewen as [maintainer](http://odoo-community.org/page/maintainer-role)
* [development_status](http://odoo-community.org/page/development-status) = Mature

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by simahawk** [comment]
@sbidoul final result seems weird, at least [here on GH](https://github.com/acsone/queue/blob/85c30d765a5bbe0283ada32b98733bf4d406e077/queue_job/README.rst):

image

**Comment by simahawk** [comment]
also, travis is red :)

**Comment by sbidoul** [comment]
>  final result seems weird

I indeed noticed the TOC bullet list does not look nice when DESCRIPTION.rst itself ends with a bullet list.

I'll add a bold **Table of contents** line in the template as a separator.

Travis is going to be just fine. For some reason it attempted to build a wrong branch.

**Comment by sbidoul** [comment]
@simahawk if you have any other comment on the overall readme structure, shoot now :)

**Comment by sbidoul** [comment]
Updated. See also https://github.com/OCA/maintainer-tools/pull/343

**Comment by pedrobaeza** [comment]
About this, AFAIK @lmignon is also an active maintainer, isn't it?

**Review by simahawk — APPROVED** [review]

**Comment by guewen** [comment]
Very nice, thanks!

> development_status = Mature

So I created #72 to be in line with the requirements :)

**Review by guewen — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Waiting for my question about maintainers before merging.

**Comment by sbidoul** [comment]
@pedrobaeza it's obvious that guewen has the maintainer role here, that's why I proposed it as a way to kickstart the process of adding maintainers somewhere. If @guewen wants help and @lmignon wants to do it I'm fine with that of course, but that is up to them to coordinate and propose, as described in the [maintainer role](http://odoo-community.org/page/maintainer-role) document.

File: queue_job/readme/CONFIGURE.rst
Comment (guewen) [inline queue_job/readme/CONFIGURE.rst:25]: That's out of scope and not new, but the basic configuration shown here can block all the HTTP workers. I think we should put `channels = root:2` or put more HTTP workers. (I can do it in a subsequent PR)

Comment (pedrobaeza) [reply queue_job/readme/CONFIGURE.rst:25]: Then it's good to change this at the same time I think

Comment (sbidoul) [reply queue_job/readme/CONFIGURE.rst:25]: Yes the queue_job documentation needs some love. I did not touch the content here.

**Comment by pedrobaeza** [comment]
@sbidoul OK, I though this was the case for a shared maintainer role (and more as this module is huge).

File: queue_job/readme/CONFIGURE.rst
Comment (sbidoul) [reply queue_job/readme/CONFIGURE.rst:25]: BTW with workers = 4 it would even be root:1, because one wkhtmlpdf report needs 2 workers so we need at least 3 free to generate a report and keep the UI working while the report runs.

**Review by lmignon — APPROVED** [review]
Great!!! 
@pedrobaeza I consider myself as a contributor and reviewer. Guewen is the maintainer.

File: queue_job/readme/CONFIGURE.rst
Comment (guewen) [reply queue_job/readme/CONFIGURE.rst:25]: Right. I propose to merge this PR without any changes to the content.

Comment (pedrobaeza) [reply queue_job/readme/CONFIGURE.rst:25]: OK, merging now and waiting for later PRs for these considerations.

**Comment by sbidoul** [comment]
BTW I prefer to have many maintainers very active on few modules each, rather than few maintainers risking burnout on many modules. We'll need to launch a campaign to have maintainers declare themselves.

**Comment by pedrobaeza** [comment]
@sbidoul OK, let's try! On the code sprint we can also make a scan over repositories for this.

**Comment by sbidoul** [comment]
Oops, forgot CONTRIBUTORS.rst, added at 0a9a13dba39d0fc962b8d8c5337e18664b8a8bc5

**Pull request merged**
